notebook/automl_classification.ipynb

```diff
@@ -39,7 +39,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "%pip install flaml[notebook]==1.1.2"
+    "%pip install flaml[notebook]==1.1.2 matplotlib openml==0.10.2"
```

Comment:
```suggestion
    "%pip install flaml[automl]==1.2.3 matplotlib openml==0.10.2"
```
And the `from flaml.data import CLASSIFICATION` in section  "4. Customized Learner" should be updated to `from flaml.automl.task.task import CLASSIFICATION`

Reply:
Also, maybe it's a good time to update and check all the notebooks as mentioned in #1012

We need to test each of the notebooks with a new virtual env to make sure it works as expected.

Reply:
> Also, maybe it's a good time to update and check all the notebooks as mentioned in #1012
> 
> We need to test each of the notebooks with a new virtual env to make sure it works as expected.

That'll be nice. Some notebooks require GPUs though, which will be hard to automate.